Engine/Code/Engine/Maths/Random.cpp

```diff
@@ -17,6 +18,11 @@ float Random::GetRandomInRange()
 	return GetRandomFloatInRange(min, max);
 }
 
+void Random::SetSeed()
+{
+	srand(unsigned int(time(0)));
```

Comment:
Why is seed time based for all we care pick a default seed like 1234 and pass in a value in case you want to have a different seed on game reset could just be an integer on the app that you increment and pass to the seed on reset

Reply:
done